### PR TITLE
Merging Project 3 code into main

### PR DESCRIPTION
**Pull request by michedmnt**
** I am aware my Pseudo Code is missing**
**I found my Pseudo Code overbearing, I need to put it in more strategically, and easier for someone to view - I will reassess**

MVP: 
- Add and delete plan saved in firebase ✅

STRETCH GOALS: 
- Add a Notes & Goals Section

FURTHER STRETCH GOALS: 
- Give it range of time slots (xx time to xx time)
- Once it passes the set time, it will automatically cross itself out

Original Pseudo Code
PseudoCode

Approved Changes for Goals
ApprovedChanges

### CODE REVIEW

File: src/partials/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@font-face {
+  font-family: "Pacifico";
+  src: url(./fonts/Pacifico/Pacifico-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url(./fonts/Montserrat/Montserrat-Italic-VariableFont_wght.ttf), url(./fonts/Montserrat/Montserrat-VariableFont_wght.ttf);
```
Comment (pocket0079) [inline src/partials/_typography.scss:10]: Great looking site!! Maybe use gitignore for the fonts that are not being used

**Comment by simpliweb** [comment]
create a footer!
everything else looks awesom! :)

File: src/partials/_typography.scss (hunk 1 above)
Comment (michedmnt) [reply src/partials/_typography.scss:10]: awesome feedback! thanks James! 😁

**Comment by michedmnt** [comment]
> 

Thank you Jie Eun!  ✨

**Pull request merged**